tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -11425,7 +11415,11 @@ st::ShaderOpTest::TShaderCallbackFn MakeShaderReplacementCallback(
       VERIFY_SUCCEEDED(hlsl::IsValidDxilContainer(pContainerHeader, compiledShader->GetBufferSize()));
       hlsl::DxilPartHeader *pPartHeader = hlsl::GetDxilPartByType(
           pContainerHeader, hlsl::DxilFourCC::DFCC_RootSignature);
-      VERIFY_IS_NOT_NULL(pPartHeader);
+      if (!pPartHeader) {
```

Comment:
I meant that this condition can be used instead of `if (!bRootSigInXML)`, so that condition goes away, and the pBuilder doesn't need to be used, so that code can be moved after this branch.

Like, after removing the `if (!bRootSigInXML)`, the braces surrounding this code, and moving builder init, you basically have:
```
// Find root signature part in container
...
if (!pPartHeader) {
  // No root signature to copy, use the assembledShader.
  *ppShaderBlob = assembledShader.Detach();
  return;
}

CComPtr<IDxcContainerBuilder> pBuilder;
VERIFY_SUCCEEDED(dllSupport.CreateInstance(CLSID_DxcContainerBuilder, &pBuilder));
VERIFY_SUCCEEDED(pBuilder->Load(assembledShader));

// Wrap root signature in blob
...
```

Then there's no need for the bRootSigInXML parameter.  It just copies the root sig if there is one.